src/session_pool/session.js

```diff
@@ -200,9 +200,14 @@ export class Session {
      * @param response
      */
     putResponse(response) {
```

Comment:
Any chance to improve the naming and API of these functions? :) You know, otherwise there is a good chance it will end up this way forever.

Reply:
Ok, let's do it now. I guess we have enough feedback and since these functions won't be used by users in `CheerioCrawler` nor `PuppeteerCrawler` we don't need to wait any longer :).

Reply:
#555